core/src/Streamly/Internal/Data/MutArray/Type.hs

```diff
@@ -68,17 +68,17 @@ module Streamly.Internal.Data.MutArray.Type
     -- | Get a subarray without copying
     , unsafeGetSlice -- XXX unsafeSliceAtLen
     , getSlice -- XXX sliceAtLen
```

Comment:
Rename the above two as well. 

Also, do not deprecate if the API was introduced in this release itself. You can check the packdiff output.